src/main/resources/aeroplan_distances.csv

```diff
@@ -620,6 +620,7 @@ YQB,YYC,,1926
 YQB,YYZ,454,453
 YQM,YYZ,749,748
 YQR,YVR,828,827
+YQR,YXE,,148
```

Comment:
X comes after W 😉

Reply:
A, B, C, D, E, F, G, Adam doesn't know the rest :(